test/models/filter_test.rb

```diff
@@ -12,11 +12,11 @@ class FilterTest < ActiveSupport::TestCase
     create(:developer, tags: [tag_1, tag_2])
     dev_3 = create(:developer, tags: [tag_2, tag_3])
 
-    params = {}
-    params["Sort By"] = "Rate: Descending"
-    params["filter"] = {
-      tag_1.name => "0", tag_2.name => "1", tag_3.name => "1"
-    }
+    params = {
+               "filter" => {"sort" => "rate desc" },
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Space inside { missing.